projects/novo-elements/src/elements/query-builder/condition-builder/condition-builder.component.ts

```diff
@@ -43,26 +50,67 @@ export class ConditionOperatorOutlet implements QueryFilterOutlet {
   selector: 'novo-condition-builder',
   templateUrl: './condition-builder.component.html',
   styleUrls: ['./condition-builder.component.scss'],
-  providers: [{ provide: NOVO_CONDITION_BUILDER, useExisting: ConditionBuilderComponent }],
+  providers: [{ provide: NOVO_CONDITION_BUILDER, useExisting: ConditionBuilderComponent },
+    {
+      provide: QueryBuilderService,
+      deps: [NovoLabelService, [new SkipSelf(), new Optional(), QueryBuilderService]],
+      useFactory: (labelService: NovoLabelService, qbs?: QueryBuilderService) => {
```

Comment:
not a huge deal but i usually prefer writing the whole word like queryBuilderService instead of qbs for readability

Reply:
`qbs` was based on the variable used in the rest of the component. Maybe we could change it in the component variable as well if we agree it looks odd.